### DISCUSSION TITLE
API Questions

### DISCUSSION THREAD

**Discussion by edrobinson** [freelawproject/courtlistener#4670] [Q&A]
I have a couple of questions about the APIs:

1. What is the root URI for the ***absolute_url*** fields?
2.  Do all of the APIs accept a ***query string***?

Thanks for any help with  these.

**Comment by mlissner** [comment]
The root uri is www.courtlistener.com.

Hm, yes, I think all of the APIs accept filtering and sorting parameters via the query string, but this is a bit of an odd question. What's the underlying thing you're trying to do?

**Comment by edrobinson** [comment]
Thanks, Mike. Ithink I am getting close to using the APIs productively.

Regards,
Ed

On Fri, Nov 8, 2024 at 1:30 PM Mike Lissner ***@***.***>
wrote:

> The root uri is www.courtlistener.com.
>
> Hm, yes, I think all of the APIs accept filtering and sorting parameters
> via the query string, but this is a bit of an odd question. What's the
> underlying thing you're trying to do?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/freelawproject/courtlistener/discussions/4670#discussioncomment-11193150>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFCS5Z6DMRH4OWFKTWC3OLZ7UNQDAVCNFSM6AAAAABROHIGIWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMJZGMYTKMA>
> .
> You are receiving this because you authored the thread.Message ID:
> <freelawproject/courtlistener/repo-discussions/4670/comments/11193150@
> github.com>
>


-- 
Ed Robinson
Ph: 208-376-2680
